tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "files": [
+    "server/src/index.ts"
+  ],
+  "include": [
+    "server/**/*.ts"
+  ],
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es6",
+    "outDir": "dist",
+    "rootDir": "server",
+    "sourceMap": true,
```

Comment:
@danbilokha Do we need source map in production version?
I suppose this is the only one situation when are going to build app, right?

Reply:
not sure :D